rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -70,6 +73,14 @@ gs_frame::gs_frame(const QRect& geometry, const QIcon& appIcon, const std::share
 	// Change cursor when in fullscreen.
 	connect(this, &QWindow::visibilityChanged, this, &gs_frame::HandleCursor);
 
+	// Configure the mouse hide on idle timer
+	connect(&m_mousehide_timer, SIGNAL(timeout()), this, SLOT(MouseHideTimeout()));
+	m_mousehide_timer.setSingleShot(true);
+	if (m_hide_mouse_after_idletime)
+	{
+		m_mousehide_timer.start(m_hide_mouse_idletime); // we start the idle timer
```

Comment:
why is this done here?
it's already done in the showEvent

Reply:
The starting of the idle timer?
Not sure I understand the 'already done in the showEvent'.
Will showEvent trigger a mousemove event, or a change in fullscreen mode?

Reply:
when the visibility changes, we fire HandleCursor, which will in turn trigger the timer if necessary.
So there is no need to duplicate the code.

(At least that's how it should work if I'm not mistaken)

Reply:
Ok, I'll remove from here.